docs/installation.md

```diff
@@ -1,111 +1,135 @@
-# Install the bot
-This page explains how to prepare your environment for running the bot. 
-To understand how to set up the bot please read the Bot 
-[Bot configuration](https://github.com/gcarq/freqtrade/blob/develop/docs/configuration.md) 
-page.
+# Installation
+
+This page explains how to prepare your environment for running the bot.
+
+To understand how to set up the bot please read the [Bot Configuration](https://github.com/gcarq/freqtrade/blob/develop/docs/configuration.md) page.
+
+
 
 ## Table of Contents
-- [Docker Automatic Installation](#docker)
-- [Linux or Mac manual Installation](#linux--mac)
-    - [Linux - Ubuntu 16.04](#21-linux---ubuntu-1604)
-    - [Linux - Other distro](#22-linux---other-distro)
-    - [MacOS installation](#23-macos-installation)
-    - [Advanced Linux ](#advanced-linux)
-- [Windows manual Installation](#windows)
 
-# Docker
+* [Table of Contents](#table-of-contents)
+* [Automatic Installation - Docker](#automatic-installation-docker)
+* [Custom Installation](#custom-installation)
+	- [Requirements](#requirements)
+	- [Linux - Ubuntu 16.04](#linux-ubuntu-1604)
+	- [MacOS](#macos)
+	- [Windows](#windows)
+* [First Steps](#first-step)
+
+<!-- /TOC -->
+------
+
+## Automatic Installation - Docker
 
-## Easy installation
 Start by downloading Docker for your platform:
-- [Mac](https://www.docker.com/products/docker#/mac)
-- [Windows](https://www.docker.com/products/docker#/windows)
-- [Linux](https://www.docker.com/products/docker#/linux)
 
-Once you have Docker installed, simply create the config file 
-(e.g. `config.json`) and then create a Docker image for `freqtrade` 
-using the Dockerfile in this repo.
+* [Mac](https://www.docker.com/products/docker#/mac)
+* [Windows](https://www.docker.com/products/docker#/windows)
+* [Linux](https://www.docker.com/products/docker#/linux)
+
+Once you have Docker installed, simply create the config file (e.g. `config.json`) and then create a Docker image for `freqtrade` using the Dockerfile in this repo.
+
+
+### 1. Prepare the Bot
+
+#### 1.1. Clone the git repository
 
-### 1. Prepare the bot
-1. Clone the git 
 ```bash
-git clone git@github.com:gcarq/freqtrade.git
+git clone https://github.com/gcarq/freqtrade.git
 ```
-2. (Optional) Checkout the develop branch
+
+#### 1.2. (Optional) Checkout the develop branch
+
 ```bash
 git checkout develop
 ```
-3. Go into the new directory
+
+#### 1.3. Go into the new directory
+
 ```bash
 cd freqtrade
 ```
-4. Copy `config.sample` to `config.json`
+
+#### 1.4. Copy `config.json.example` to `config.json`
+
 ```bash
-cp config.json.example config.json
+cat config.json.example >> config.json
```

Comment:
We are copying a file, so I'd rather stick to `cp` rather than `cat`

Reply:
`cp` will clobber any existing file... `cat >>` will append and not overwrite... but I see both points...

Reply:
will append yes, but then what are the likelyhoods that the appended config file is valid json anymore?

Reply:
in this case I suggest to add the param -n

`cp -n config.json.example config.json`